bin/preparedb

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env click-odoo
```

Comment:
I prefer it if you set here:

```suggestion
#!/usr/bin/env python
```

Then, inside python, you import click_odoo.

Take a look into any script from click-odoo-contrib, such as https://github.com/acsone/click-odoo-contrib/blob/4fa3b8f0b4ad6a12042fffde8a54150accf79f5c/click_odoo_contrib/dropdb.py

This way, the file will be detected as python and used by linters, formatters, syntax highlighters, etc.